docker/simple-server-base.Dockerfile

```diff
@@ -37,26 +38,24 @@ RUN wget https://cache.ruby-lang.org/pub/ruby/2.7/ruby-2.7.8.tar.gz && \
 # Install a compatible version of Bundler for Ruby 2.7.8
 RUN gem install bundler -v 2.4.22
 
-# Yarn
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg -o /root/yarn-pubkey.gpg && apt-key add /root/yarn-pubkey.gpg
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list
-RUN apt-get update && apt-get install -y --no-install-recommends yarn
+# Install Yarn
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg -o /root/yarn-pubkey.gpg && apt-key add /root/yarn-pubkey.gpg && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" > /etc/apt/sources.list.d/yarn.list && \
+    apt-get update && apt-get install -y --no-install-recommends yarn
```

Comment:
Please revert this to independent `RUN` statements. The process fails if a step fails anyways. Chaining them makes them harder to debug.

The difference between this and the one up tops is that the one up tops uses the same command `apt-get`; this one is has different commands. Let's keep them separate